Sources/Plasma/NucleusLib/pnNetProtocol/Private/Protocols/Cli2Auth/pnNpCli2Auth.cpp

```diff
@@ -149,9 +149,9 @@ static const NetMsgField kAcctActivateRequestFields[] = {
 };
 
 static const NetMsgField kFileListRequestFields[] = {
-    kNetMsgFieldTransId,                // transId
-    NET_MSG_FIELD_STRING(MAX_PATH),     // directory
-    NET_MSG_FIELD_STRING(MAX_EXT),      // ext
```

Comment:
This is not the same as `MAX_PATH` btw - `MAX_EXT` is defined to 256, not 260. Also, `MAX_EXT` comes from Plasma itself and not Windows, so it's less of a problem.